python/prompt-management/temp.py

```
@@ -0,0 +1,160 @@
+import base64
+import csv
+import json
+import re
+
+import pandas as pd
+
+
+def escape_for_clickhouse(value):
```

Ruff (0.11.9)

Missing return type annotation for public function `escape_for_clickhouse`

Add return type annotation: `str`

(ANN201)